app/workers/unfilter_notifications_worker.rb

```diff
@@ -3,8 +3,18 @@
 class UnfilterNotificationsWorker
   include Sidekiq::Worker
 
-  def perform(notification_request_id)
-    @notification_request = NotificationRequest.find(notification_request_id)
+  # Earlier versions of the feature passed a `notification_request` ID
+  # If `to_account_id` is passed, the first argument is an account ID
+  def perform(notification_request_or_account_id, recipient_id = nil)
+    if to_account_id.present?
```

Comment:
Whats `to_account_id` here? Should be `recipient_id`?

Reply:
That was indeed an oversight! Fixed and added tests.